docs/using-cinder-csi-plugin.md

```diff
@@ -405,6 +405,9 @@ $ kubectl create -f manifests/cinder-csi-plugin/
 ```
 $ kubectl create -f examples/cinder-csi-plugin/inline/inline-example.yaml
 ```
+
+> NOTE: `VolumeAttributes` is optional, if it's not provided, it will default to `1Gi`
```

Comment:
@jichenjc As `capacity` value defaults to 1Gi, its better to add like 
specifying volume `capacity` is optional , if it's not provided, it will default to `1Gi`

Reply:
done, thanks~